### ISSUE TITLE
Automatically set thumbnail for non-pdf files

### ISSUE THREAD

**Issue by jillpe** [notch8/utk-hyku#438]
# Summary
Currently, thumbnails need to be manually created by editing the work. We would like to not have this extra step and for it to be created upon import. 

# Accepted Criteria
- [ ] Thumbnails for non-pdf files are created upon import

# Testing Instructions
- [ ] Import this: [rfta_audio_test_3_with_attachments.csv](https://github.com/scientist-softserv/utk-hyku/files/11853735/rfta_audio_test_3_with_attachments.csv)
- [ ] wait for all jobs to finish and observe that there is an thumbnail for the parent record

**Comment by jillpe** [comment]
related to #215

**Comment by jillpe** [comment]
related PR: https://github.com/scientist-softserv/utk-hyku/pull/435/files#diff-9a0b6467d5e8c3e0a1812ff91b5a8230de81d0589be0c35cdc6b846f40d3560b

**Comment by ShanaLMoore** [comment]
⚠ ~~QA BLOCKED by #446 ⚠ CreateRelationshipJob bug~~

QA unblocked. The issue reported was due to bad data in the CSV which caused CreateRelationshipJobs to fail, so this should not be related to or blocking this after all.

**Comment by DeonFranklin** [comment]
![Image](https://github.com/scientist-softserv/utk-hyku/assets/95306716/3abd02fe-a9e1-4633-8d0a-c290dc9408fa)



![Image](https://github.com/scientist-softserv/utk-hyku/assets/95306716/d91ebee0-cc1a-469d-ad79-40291528dcf7)



![Image](https://github.com/scientist-softserv/utk-hyku/assets/95306716/e785f70b-9290-4c66-be84-eacf26a29d74)



![Image](https://github.com/scientist-softserv/utk-hyku/assets/95306716/c95fa3a3-eaf9-44c5-a3d7-4e550c87b7cd)

**Comment by jillpe** [comment]
SoftServ QA: Pulling back for rework

CSV imported: 

[rfta_audio_test_3_with_attachments (1).csv](https://github.com/scientist-softserv/utk-hyku/files/11899428/rfta_audio_test_3_with_attachments.1.csv)



[Work URL](https://qa.utk-hyku-staging.notch8.cloud/concern/audios/b49b6d89-5953-4493-ba5a-2f26ad98339d?locale=en)

[Import URL](https://qa.utk-hyku-staging.notch8.cloud/importers/37?locale=en)

Uploaded csv and watched all of the jobs complete in sidekiq (including the thumbnail job). Not thumbnails appeared on the work show page or admin dashboard. See screenshots below

<details><summary>Screenshots</summary>


![Image](https://github.com/scientist-softserv/utk-hyku/assets/84697174/2a99a631-ba24-4f96-ae11-457f6e81f9be)



![Image](https://github.com/scientist-softserv/utk-hyku/assets/84697174/558d67b1-a0d1-4b57-95b6-3e31d29db00b)



![Image](https://github.com/scientist-softserv/utk-hyku/assets/84697174/b3647f50-64e7-4b9c-9690-c91652a9fcb4)


</details>

**Comment by ShanaLMoore** [comment]
> SoftServ QA: Pulling back for rework
> 
> CSV imported:
> 
> [rfta_audio_test_3_with_attachments (1).csv](https://github.com/scientist-softserv/utk-hyku/files/11899428/rfta_audio_test_3_with_attachments.1.csv)
> 
> [Work URL](https://qa.utk-hyku-staging.notch8.cloud/concern/audios/b49b6d89-5953-4493-ba5a-2f26ad98339d?locale=en)
> 
> [Import URL](https://qa.utk-hyku-staging.notch8.cloud/importers/37?locale=en)
> 
> Uploaded csv and watched all of the jobs complete in sidekiq (including the thumbnail job). Not thumbnails appeared on the work show page or admin dashboard. See screenshots below
> Screenshots

Until derivative rodeo is in place, audio files won't have the ability to set thumbnails. Your CSV is importing audio files so I wouldn't expect this to work right now. There are no images it can pull from. Please try this test again with:

[442.csv](https://github.com/scientist-softserv/utk-hyku/files/11899818/442.csv)

**Comment by ShanaLMoore** [comment]
cc @kirkkwang on your return could you please clarify the expectation for audio files and thumbnails?

**Comment by kirkkwang** [comment]
Hi @ShanaLMoore, the expectation for specifically audio works is that they should have thumbnails, which looks like a file with the audio logo

**Comment by ShanaLMoore** [comment]
Thanks for clarifying Kirk! that's what we were seeing but for some reason it didn't register as a thumbnail to me lol. I guess I was expecting an image/photo. ie a photo of the musician or something, which at this point didn't think was possible.

**Comment by jillpe** [comment]
SoftServ QA: Pass!
 

> SoftServ QA: Pulling back for rework

CSV imported: 

[rfta_audio_test_3_with_attachments (1).csv](https://github.com/scientist-softserv/utk-hyku/files/11899428/rfta_audio_test_3_with_attachments.1.csv)



[Work URL](https://qa.utk-hyku-staging.notch8.cloud/concern/audios/b49b6d89-5953-4493-ba5a-2f26ad98339d?locale=en)

[Import URL](https://qa.utk-hyku-staging.notch8.cloud/importers/37?locale=en)

Uploaded csv and watched all of the jobs complete in sidekiq (including the thumbnail job). Not thumbnails appeared on the work show page or admin dashboard. See screenshots below

<details><summary>Screenshots</summary>


![Image](https://github.com/scientist-softserv/utk-hyku/assets/84697174/2a99a631-ba24-4f96-ae11-457f6e81f9be)



![Image](https://github.com/scientist-softserv/utk-hyku/assets/84697174/558d67b1-a0d1-4b57-95b6-3e31d29db00b)



![Image](https://github.com/scientist-softserv/utk-hyku/assets/84697174/b3647f50-64e7-4b9c-9690-c91652a9fcb4)


</details>

This did pass, we did not realize the thumbnail was the music notes

**Comment by kirkkwang** [comment]
@ShanaLMoore I thought I heard something about eventually using a more curated thumbnail for their audio files, but would probably be a separate ticket to get that working

@jillpe does that sound familiar?
